microservices/selfcare-onboarding-consumer/att/values.yaml

```
@@ -6,15 +6,6 @@ serviceAccount:
 
 service:
   create: true
```

```suggestion
  create: false
```
If it's just a consumer we can remove the K8s Service